zkevm-circuits/src/rlp_circuit_fsm.rs

```diff
@@ -781,6 +808,28 @@ impl<F: Field> RlpCircuitConfig<F> {
             |meta| meta.query_advice(format, Rotation::next()),
         );
 
+        // constrain diff belong to range256 table for each comparator
+        let mut diffs = vec![];
+        diffs.extend(byte_value_gte_0x80.lt_chip.config().diff);
+        diffs.extend(byte_value_gte_0xb8.lt_chip.config().diff);
+        diffs.extend(byte_value_gte_0xc0.lt_chip.config().diff);
+        diffs.extend(byte_value_gte_0xf8.lt_chip.config().diff);
+        diffs.extend(byte_value_lte_0x80.lt_chip.config().diff);
+        diffs.extend(byte_value_lte_0xb8.lt_chip.config().diff);
+        diffs.extend(byte_value_lte_0xc0.lt_chip.config().diff);
+        diffs.extend(byte_value_lte_0xf8.lt_chip.config().diff);
+        diffs.extend(tidx_lte_tlength.lt_chip.config().diff);
+        diffs.extend(mlength_lte_0x20.lt_chip.config().diff);
+
```

Comment:
The new comparator `tlength_lte_mlength` might also need the `lt_chip.diff` columns to be range checked.

Reply:
fixed in https://github.com/scroll-tech/zkevm-circuits/pull/572/commits/baa4a7be0a68e8a3da63b6b2caeb0902d81ec90f.